seatunnel-engine/seatunnel-engine-core/src/main/java/org/apache/seatunnel/engine/core/parse/JobConfigParser.java

```diff
@@ -221,12 +221,12 @@ static String createSourceActionName(int configIndex, String pluginName) {
         return String.format("Source[%s]-%s", configIndex, pluginName);
     }
 
-    static String createSinkActionName(int configIndex, String pluginName) {
-        return String.format("Sink[%s]-%s", configIndex, pluginName);
-    }
-
     static String createSinkActionName(int configIndex, String pluginName, String table) {
-        return String.format("Sink[%s]-%s-%s", configIndex, pluginName, table);
+        if (table != null && !table.isEmpty()) {
+            return String.format("Sink[%s]-%s", configIndex, pluginName, table);
+        } else {
+            return String.format("Source[%s]-%s", configIndex, pluginName);
+        }
     }
```

Comment:
Hi @anirudh-hegde , thanks for created this PR, but I think this change has no obvious meaning and is more confusing for callers.

Reply:
> Hi @anirudh-hegde , thanks for created this PR, but I think this change has no obvious meaning and is more confusing for callers.

I think using if condition instead of writing two functions is a better practice.